xds/src/main/java/io/grpc/xds/XdsClientImpl.java

```diff
@@ -60,13 +60,18 @@
 import java.util.Set;
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 import javax.annotation.Nullable;
 
 /**
  * XdsClient implementation for client side usages.
  */
 final class XdsClientImpl extends XdsClient implements XdsResponseHandler, ResourceStore {
 
+  static boolean logXdsNodeId = Boolean.parseBoolean(System.getenv("GRPC_LOG_XDS_NODE_ID"));
```

Comment:
Make both of these private.

This should be named with the LOG_XDS_NODE_ID style as it is immutable. (Logger is not immutable, so it uses the normal naming.)

Reply:
Done. PTAL!